respondd.py

```diff
@@ -51,6 +54,48 @@ def handle(self):
 
     return ResponddUDPHandler
 
+
+def listen(iface, group, port, directory, env):
+    # multicast
+    print("binding to {0}%{1} port {2}...".format(
+        group, iface, port)
+    )
+    mcast_server = socketserver.ThreadingUDPServer(
+        (group, port, 0, socket.if_nametoindex(iface)),
+        get_handler(directory, env)
+    )
+
+    group_bin = socket.inet_pton(socket.AF_INET6, group)
+    mreq = group_bin + struct.pack('@I', socket.if_nametoindex(iface))
```

Comment:
If you start respondd with an interface that doesn't (yet) exist, this call to `socket.if_nametoindex` will raise an uncaught exception. But as we (re)bind to (specified) interfaces appearing later on, this could just as well be caught and return an empty thread list.